extensions/roc-plugin-dredd/src/config/roc.config.js

```diff
@@ -1 +1,16 @@
-export default {};
+export default {
+    settings: {
+        test: {
+            dredd: {
```

Comment:
Does it make sense to expose all Dredd options through Roc, mainly based on a signal to noise ratio. Are all settings something that the user of this plugin might want to modify and especially set through using command line options.

Another option to minimise the noise but still keep the full configurable power is to use `__raw`.

https://github.com/rocjs/roc/blob/master/docs/Settings.md#escape-hatch

See related discussion here: https://github.com/rocjs/roc-plugin-browsersync/issues/6